src/Files.App/Views/Properties/SecurityAdvancedPage.xaml

```diff
@@ -547,72 +412,11 @@
 								x:Name="InheritanceFlagsHumanizedTextBlock"
 								Grid.Column="5"
 								VerticalAlignment="Center"
-								x:Load="{x:Bind IsEditable, Mode=OneWay, Converter={StaticResource BoolNegationConverter}}"
 								Text="{x:Bind InheritanceFlagsHumanized, Mode=OneWay}"
 								TextTrimming="CharacterEllipsis"
 								TextWrapping="NoWrap"
 								Visibility="{x:Bind IsFolder, Mode=OneWay}" />
 
-							<!--  Applies To Changer DropDown  -->
-							<DropDownButton
-								x:Name="InheritanceFlagsEditDropDownButton"
-								Grid.Column="5"
-								Padding="0,6"
-								HorizontalAlignment="Stretch"
-								VerticalAlignment="Center"
-								HorizontalContentAlignment="Left"
-								x:Load="{x:Bind IsEditable, Mode=OneWay}"
-								Background="Transparent"
-								BorderThickness="0"
-								Visibility="{x:Bind IsFolder, Mode=OneWay}">
-
-								<DropDownButton.Resources>
-									<ResourceDictionary>
-										<StaticResource x:Key="ButtonBackgroundPointerOver" ResourceKey="SystemControlTransparentBrush" />
-										<StaticResource x:Key="ButtonBackgroundPressed" ResourceKey="SystemControlTransparentBrush" />
-										<StaticResource x:Key="ButtonBackgroundDisabled" ResourceKey="SystemControlTransparentBrush" />
-									</ResourceDictionary>
-								</DropDownButton.Resources>
-
-								<DropDownButton.Content>
-									<TextBlock Text="{x:Bind InheritanceFlagsHumanized, Mode=OneWay}" TextTrimming="CharacterEllipsis" />
-								</DropDownButton.Content>
-
-								<DropDownButton.Flyout>
```

Comment:
If the strings are no longer being used, can you remove them from the resw file?

Reply:
No strings have been unused.

Reply:
Removed unused strings. Thank you for pointint it out!